### PR TITLE
Fix forms puppet uwp

### PR DESCRIPTION
**Pull request by guperrot**

### CODE REVIEW

**Comment by guperrot** [comment]
@dhei compilation fixed but I don't receive toasts, please help fixing Forms Puppet, this was totally overlooked...

**Comment by guperrot** [comment]
Because the UWP code was not shared with portable, the callback used by xamarin is missing in uwp assembly.

People calling `Push.PushNotificationReceived += ...` in portable will crash in UWP with

```
```System.TypeLoadException occurred
  HResult=0x80131522
  Message=Could not load type 'Microsoft.Azure.Mobile.Push.PushNotificationReceivedEventArgs' from assembly 'Microsoft.Azure.Mobile.Push, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
  Source=<Cannot evaluate the exception source>
  StackTrace:
   at Contoso.Forms.Demo.App..ctor() in D:\git\mobile-center-sdk-dotnet\Apps\Contoso.Forms.Demo\Contoso.Forms.Demo\App.xaml.cs:line 27
   at Contoso.Forms.Demo.UWP.MainPage..ctor() in D:\git\mobile-center-sdk-dotnet\Apps\Contoso.Forms.Demo\Contoso.Forms.Demo.UWP\MainPage.xaml.cs:line 13
   at Contoso.Forms.Demo.UWP.Contoso_Forms_Demo_UWP_XamlTypeInfo.XamlTypeInfoProvider.Activate_4_MainPage() in D:\git\mobile-center-sdk-dotnet\Apps\Contoso.Forms.Demo\Contoso.Forms.Demo.UWP\obj\x86\Debug\XamlTypeInfo.g.cs:line 207
   at Contoso.Forms.Demo.UWP.Contoso_Forms_Demo_UWP_XamlTypeInfo.XamlUserType.ActivateInstance() in D:\git\mobile-center-sdk-dotnet\Apps\Contoso.Forms.Demo\Contoso.Forms.Demo.UWP\obj\x86\Debug\XamlTypeInfo.g.cs:line 427```
```

(Just tested in demo with stable nugets)

File: Tests/Microsoft.Azure.Mobile.Push.Test.Windows/PushTest.cs [hunk 1]
```diff
@@ -21,11 +21,11 @@ public void InstanceIsNotNull()
         [TestMethod]
         public void GetEnabled()
         {
-            Push.Enabled = false;
```
Comment (guperrot) [inline Tests/Microsoft.Azure.Mobile.Push.Test.Windows/PushTest.cs]: we should test the public API actually, make test project reference shared one.

Comment (dhei) [inline Tests/Microsoft.Azure.Mobile.Push.Test.Windows/PushTest.cs]: good catch. fixed now.

**Pull request merged**